js_modules/dagit/packages/core/src/app/AppTopNav.tsx

```diff
@@ -58,7 +58,18 @@ export const AppTopNav: React.FC<Props> = ({
             shortcutLabel="⌥3"
             shortcutFilter={(e) => e.code === 'Digit3' && e.altKey}
           >
-            <TopNavLink to="/instance" data-cy="AppTopNav_StatusLink">
+            <TopNavLink
+              to="/instance"
+              data-cy="AppTopNav_StatusLink"
+              isActive={(_, location) => {
+                const {pathname} = location;
+                return (
```

Comment:
This feels a bit gross but I don't have any other ideas :( I feel like there might be a way to make this the "default active" if the others aren't active, but that's not quite what we want either. This seems good for now!